src/rgb/mod.rs

```diff
@@ -0,0 +1,68 @@
+//!RGB types, spaces and standards.
+
+use num::Float;
+
+use Yxy;
+use white_point::WhitePoint;
+use pixel::TransferFn;
+use std::any::Any;
+
+pub use self::linear::{LinRgb, LinRgba};
+pub use self::nonlinear::{Rgb, Rgba};
+
+pub mod standards;
+mod linear;
+mod nonlinear;
+
+///Nonlinear sRGB.
+pub type Srgb<T = f32> = Rgb<standards::Srgb, T>;
+///Nonlinear sRGB with an alpha component.
+pub type Srgba<T = f32> = Rgba<standards::Srgb, T>;
+
+///Linear sRGB.
+pub type LinSrgb<T = f32> = LinRgb<standards::Srgb, T>;
+///Linear sRGB with an alpha component.
+pub type LinSrgba<T = f32> = LinRgba<standards::Srgb, T>;
+
+///An RGB space and a transfer function.
+pub trait RgbStandard {
```

Comment:
Should the RgbStandard have the WhitePoint type as well? The usecase I see for it is supporting non standard white points like ICC Srgb D50 \<SrgbTranfFn, SrgbSpace, D50\>.